doc/modules/cassandra/pages/managing/operating/repair.adoc

```diff
@@ -29,10 +29,18 @@ for syncing up missed writes, but it doesn't protect against things like
 disk corruption, data loss by operator error, or bugs in Cassandra. For
 this reason, full repairs should still be run occasionally.
 
-== Usage and Best Practices
+== Automated Repair Scheduling
 
-Since repair can result in a lot of disk and network io, it's not run
-automatically by Cassandra. It is run by the operator via nodetool.
+Since repair can result in a lot of disk and network io, it has
+traditionally not been run automatically by Cassandra.
+
+In the latest version of Cassandra, a new feature called
+xref:managing/operating/auto_repair.adoc[auto repair] was introduced to
+allow Cassandra to run repairs automatically on a schedule.
+
+== Submitting Repairs Using Nodetool
+
+Repairs can also be run by the operator via nodetool.
```

Comment:
Should we explicitly clarify that the above mentioned new feature does not interfere with the traditional *nodetool repair* functionality, otherwise some people might get confused that this new feature is breaking the traditional *nodetool repair*